### PR TITLE
mon/PGMap: call blocked requests ERR not WARN

### PR DESCRIPTION
**Pull request by liewegas**
Signed-off-by: Sage Weil <sage@redhat.com>

### CODE REVIEW

**Comment by gregsfortytwo** [comment]
I'm concerned that it's pretty common to get slow requests during peering and recovery processes on overloaded clusters. This is going to significantly turn up the amount of "errors" sys admins see and have to react to. Could we at least only make it an error when they stick around for a threshold higher than the one where we mark them slow?

**Comment by liewegas** [comment]
how about replacing mon_osd_max_op_age with mon_osd_warn_op_age and mon_osd_err_op_age (with an associated release note)?

**Comment by gregsfortytwo** [comment]
Or maybe just make the err_op_age a multiplier on top of the warn age?

**Comment by liewegas** [comment]
http://pulpito.ceph.com/sage-2017-06-06_21:54:14-rados-wip-sage-testing-distro-basic-smithi/

File: PendingReleaseNotes [hunk 1]
```diff
@@ -137,3 +137,11 @@
   to zero will effectively disable the health check.
 * The "ceph mds tell ..." command has been removed.  It is superceded
   by "ceph tell mds.<id> ..."
+
+12.1.0
+------
+
+* The ``mon_osd_max_op_age`` option has been renamed to ``mon_osd_warn_op_age``,
+  to indicate we generate a warning at this age.  There is also a new
+  ``mon_osd_err_op_age_multiple`` that is a expressed as a multitple of
```
Comment (tchaikov) [inline PendingReleaseNotes]: s/mon_osd_err_op_age_multiple/mon_osd_err_op_age_ratio/

File: src/common/config_opts.h [hunk 2]
```diff
@@ -288,7 +288,8 @@ OPTION(mon_osd_down_out_interval, OPT_INT, 600) // seconds
 OPTION(mon_osd_down_out_subtree_limit, OPT_STR, "rack")   // smallest crush unit/type that we will not automatically mark out
 OPTION(mon_osd_min_up_ratio, OPT_DOUBLE, .3)    // min osds required to be up to mark things down
 OPTION(mon_osd_min_in_ratio, OPT_DOUBLE, .75)   // min osds required to be in to mark things out
-OPTION(mon_osd_max_op_age, OPT_DOUBLE, 32)     // max op age before we get concerned (make it a power of 2)
+OPTION(mon_osd_warn_op_age, OPT_DOUBLE, 32)     // max op age before we generate a warning (make it a power of 2)
+OPTION(mon_osd_err_op_age_ratio, OPT_DOUBLE, 2)  // when to generate an error, as multiple of mon_osd_warn_op_age
```
Comment (jdurgin) [inline src/common/config_opts.h]: should add to the config ref docs

**Comment by liewegas** [comment]
fixed!

**Pull request merged**